src/sentry/static/sentry/app/views/settings/project/projectFilters/index.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+
+import {t} from '../../../../locale';
+import GroupTombstones from './groupTombstones';
+import ProjectFiltersChart from './projectFiltersChart';
+import ProjectFiltersSettings from './projectFiltersSettings';
+import SentryTypes from '../../../../proptypes';
+import SettingsPageHeader from '../../components/settingsPageHeader';
+import TextBlock from '../../components/text/textBlock';
+
+class ProjectFilters extends React.Component {
+  static contextTypes = {
+    organization: SentryTypes.Organization,
+    project: SentryTypes.Project,
+  };
+
+  constructor(...args) {
+    super(...args);
+    this.state = {
+      activeSection: 'data-filters',
+    };
+  }
+
+  setProjectNavSection = section => {
+    this.setState({
+      activeSection: section,
+    });
+  };
+
+  render() {
+    let {organization, project} = this.context;
+    let {orgId, projectId} = this.props.params;
+    let {activeSection} = this.state;
+    if (!project) return null;
+
+    let features = new Set(project.features);
+
+    return (
+      <div>
+        <SettingsPageHeader title={t('Inbound Data Filters')} />
+        <TextBlock>
+          {t(
+            'Filters allow you to prevent Sentry from storing events in certain situations. Filtered events are tracked separately from rate limits, and do not apply to any project quotas.'
+          )}
+        </TextBlock>
+
+        <div>
+          <ProjectFiltersChart params={this.props.params} />
+
+          {features.has('discard-groups') && (
```

Comment:
I was pretty sure we implemented some kind of call to action on this feature if you didn't have it enabled (cc @macqueen?).

Reply:
I think there's a CTA for custom inbound filters, is that what you're thinking of?

Reply:
Yeah. It should be for discarded issues (discard & delete) too. Something to revisit.